src/controllers/verifyCommand.ts

```diff
@@ -13,7 +13,6 @@ export async function verifyCommand(
   env: env
 ) {
   const token = await generateUniqueToken();
-
```

Comment:
Nit: we can omit this unintended alteration.

Reply:
added it in last commit